### PR TITLE
Feature/publish

### PR DESCRIPTION
**Pull request by garyluu**
- Enabling publish button for my tools page

### CODE REVIEW

File: src/app/container/container.component.ts [hunk 1]
```diff
@@ -28,20 +32,22 @@ export class ContainerComponent extends Tool {
...
-              toolService: ToolService,
-              communicatorService: CommunicatorService,
-              providerService: ProviderService,
-              router: Router,
-              workflowService: WorkflowService,
-              containerService: ContainerService) {
+    private dateService: DateService,
+    private imageProviderService: ImageProviderService,
+    private listContainersService: ListContainersService,
+    private updateContainer: ContainerService, private containerWebService: ContainerWebService,
```
Comment (denis-yuen) [inline src/app/container/container.component.ts]: this formatting seems odd

Comment (garyluu) [inline src/app/container/container.component.ts]: My IDE automatically puts 2 space indent if it's a new line

Comment (denis-yuen) [inline src/app/container/container.component.ts]: I more meant the two variables on one line

Comment (garyluu) [inline src/app/container/container.component.ts]: oh, i've fixed that in my new branch

**Pull request merged**